html/captive-portal/lib/captiveportal/PacketFence/Controller/TLSProfile.pm

```diff
@@ -0,0 +1,242 @@
+package captiveportal::PacketFence::Controller::TLSProfile;
+use Moose;
+use namespace::autoclean;
+use WWW::Curl::Easy;
+use Crypt::OpenSSL::PKCS12;
+use pf::log;
+use pf::config;
+use pf::util;
+use pf::node;
+#use pf::web qw(i18n ni18n i18n_format render_template);
+use pf::web::constants;
+use List::MoreUtils qw(uniq any);
+#use Readonly;
+#use POSIX;
+use pf::authentication;
+use HTML::Entities;
+#use URI::Escape::XS qw(uri_escape);
+use pf::web;
+use File::Basename;
+use Email::Valid;
+
```

Comment:
I taught that by here you would have imported pf::zammit::dinde since it seems all the packages are getting imported